src/app/state/api.js

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable import/prefer-default-export */
 
+const path = '__BASE_DOMAIN__';
+const url = `${path}/api/v1/data/`;
```

Comment:
You could also do the following, right, and skip the extra variable?
`const url = '__BASE_DOMAIN__/api/v1/data/';

Reply:
I suppose so :)